installers/olm/description.md

```diff
@@ -32,6 +32,7 @@ Kubernetes, providing a seamless cloud native Postgres solution to keep your dat
 - **Affinity and Tolerations**: Have your PostgreSQL clusters deployed to [Kubernetes Nodes][k8s-nodes] of your preference.
   Set your [pod anti-affinity][k8s-anti-affinity], node affinity, Pod tolerations and more rules to customize your deployment topology!
 - **PostgreSQL Major Version Upgrades**: Perform a [PostgreSQL major version upgrade][major-version-upgrade] declaratively.
+- **Improved pgAdmin Configuration**: Declaratively deploy [pgAdmin][pgadmin] instances via a customer resource per namespace or per PostgresCluster.
```

Comment:
```suggestion
- **Improved pgAdmin Configuration**: Declaratively deploy [pgAdmin][pgadmin] instances via a custom resource per namespace or per PostgresCluster.
```

Reply:
:+1: Thanks for the catch, I'll push that change.